src/DetailsView/components/save-assessment-factory.tsx

```diff
@@ -20,9 +22,15 @@ export function getSaveButtonForAssessment(props: SaveAssessmentFactoryProps): J
     const assessmentData = props.deps.assessmentDataFormatter.formatAssessmentData(
         props.assessmentStoreData.assessments,
     );
+
+    const currentDate = new Date();
+    const fileDate = props.deps.fileNameBuilder.getDateSegment(currentDate);
+    const targetPageTitle = props.assessmentStoreData.persistedTabInfo.title;
+    const fileTitle = props.deps.fileNameBuilder.getTitleSegment(targetPageTitle);
+    const fileName = `SavedAssessment_<${fileDate}>_${fileTitle}.a11yextension`;
```

Comment:
`a11yextension` matches the spec, but as I think about this, I think we might want to update the spec here; I think it would makes sense for us to use something that's a little more specific to our product, since we don't intend to support this format for use with other a11y tools besides AI4Web. @ferBonnin, thoughts on using something a little more specific? Maybe `.aiwebassessment`?

Reply:
agree, per the spec in the feature the file extension should be` *.a11yassessment`

Reply:
@ferBonnin and I discussed further offline, let's go with `.a11ywebassessment`

Reply:
oops. that was my bad, I accidentally transposed the second half of the file extension. thank you both!